jax/_src/export/serialization_generated.py

```diff
@@ -21,20 +21,21 @@
 from flatbuffers.compat import import_numpy
 np = import_numpy()
 
-class PyTreeDefKind:
+class PyTreeDefKind(object):
```

Comment:
Huh, why would flatbuffers add an `object` base class? It's implicit in Python 3.